decidim-conferences/app/helpers/decidim/conferences/conferences_helper.rb

```diff
@@ -5,6 +5,10 @@ module Conferences
     # Helpers related to the Conferences layout.
     module ConferencesHelper
       include Decidim::ResourceHelper
+
+      def participatory_spaces_for_conference(conference_participatory_spaces)
+        cell("decidim/conferences/participatory_spaces", conference_participatory_spaces)
```

Comment:
I think you can safely replace the method call with the `cell "blabla"` one, so we can remove this whole file